homeassistant/components/zwave_js/config_flow.py

```diff
@@ -430,7 +431,7 @@ async def async_step_manual(
             step_id="manual", data_schema=get_manual_schema(user_input), errors=errors
         )
 
-    async def async_step_hassio(self, discovery_info: dict[str, Any]) -> FlowResult:
+    async def async_step_hassio(self, discovery_info: HassioServiceInfo) -> FlowResult:
         """Receive configuration from add-on discovery info.
 
         This flow is triggered by the Z-Wave JS add-on.
```

Comment:
Looks like this needs to access `.config`

Reply:
nm Martin is already reviewing this one

https://github.com/home-assistant/core/pull/60824#discussion_r761477160